src/spyglass/lfp/v1/lfp.py

```diff
@@ -85,7 +81,8 @@ def make(self, key):
         # user with those from the raw data
         orig_key = copy.deepcopy(key)
         orig_key["interval_list_name"] = key["target_interval_list_name"]
-        user_valid_times = (IntervalList() & orig_key).fetch1("valid_times")
+        user_valid_times = (IntervalList() & orig_key).fetch_interval()
```

Comment:
I think there are equivalent `fetch1("valid_times")` in decoding v1? Or am I not understanding how this works?

Reply:
`fetch1("valid_times")` will still behave the same way, fetching the values and not an `Interval` object. I only replaced `fetch1("valid_times")` in cases where there was some subsequent operation like `interval_list_contains` performed